plugins/destination_testing.go

```diff
@@ -0,0 +1,246 @@
+package plugins
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cloudquery/plugin-sdk/cqtypes"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/google/uuid"
+	"github.com/rs/zerolog"
+)
+
+// TestTable returns a table with columns of all type. useful for destination testing purposes
+func TestTable() *schema.Table {
+	return &schema.Table{
+		Name:        "cq_test_table",
+		Description: "Test table",
+		Columns: schema.ColumnList{
+			schema.CqIDColumn,
+			schema.CqParentIDColumn,
+			{
+				Name: "bool",
+				Type: schema.TypeBool,
+			},
+			{
+				Name: "int",
+				Type: schema.TypeInt,
+			},
+			{
+				Name: "float",
+				Type: schema.TypeFloat,
+			},
+			{
+				Name: "uuid",
+				Type: schema.TypeUUID,
+			},
+			{
+				Name: "text",
+				Type: schema.TypeString,
+			},
+			{
+				Name: "bytea",
+				Type: schema.TypeByteArray,
+			},
+			{
+				Name: "text_array",
+				Type: schema.TypeStringArray,
+			},
+			{
+				Name: "int_array",
+				Type: schema.TypeIntArray,
+			},
+			{
+				Name: "timestamp",
+				Type: schema.TypeTimestamp,
+			},
+			{
+				Name: "json",
+				Type: schema.TypeJSON,
+			},
+			{
+				Name: "uuid_array",
+				Type: schema.TypeUUIDArray,
+			},
+			// {
+			// 	Name: "inet",
+			// 	Type: schema.TypeInet,
+			// },
+			// {
+			// 	Name: "inet_array",
+			// 	Type: schema.TypeInetArray,
+			// },
+			// {
+			// 	Name: "cidr",
+			// 	Type: schema.TypeCIDR,
+			// },
+			// {
+			// 	Name: "cidr_array",
+			// 	Type: schema.TypeCIDRArray,
+			// },
```

Comment:
Should we uncomment these for testing purposes?

Reply:
added back